solutions/quickstart/outputs.tf

```
@@ -62,3 +62,28 @@ output "service_credential_secret_groups" {
   description = "Service credential secret groups"
   value       = length(local.service_credential_secrets) > 0 ? module.secrets_manager_service_credentials[0].secret_groups : null
 }
+
+output "next_steps_text" {
+  value       = "Your Event Streams Instance is ready."
+  description = "Next steps text"
+}
+
+output "next_step_primary_label" {
+  value       = "Go to Event Streams Instance"
+  description = "Primary label"
+}
+
+output "next_step_primary_url" {
+  value       = "https://cloud.ibm.com/services/messagehub/${urlencode(module.event_streams.crn)}?paneId=manage"
+  description = "Primary URL for the IBM Cloud Logs instance"
```

```suggestion
  description = "Primary URL"
```
Event Streams not Cloud logs